### DISCUSSION TITLE
The site shows USA for visitor, but I am in the UK

### DISCUSSION THREAD

**Discussion by davestewart** [plausible/analytics#854] [Anything Else]
## Prerequisites
- [x] I have searched open and closed issues to make sure that the bug has not yet been reported.

## Bug report

I set up the site for the first time, went to the URL and checked the dashboard. It shows visits as all being from the USA.

![image](https://user-images.githubusercontent.com/132681/111554995-5add9700-877f-11eb-9ccd-b4ee05e803a6.png)

**Expected behavior**

Should show me from the UK.

![image](https://user-images.githubusercontent.com/132681/111554937-3681ba80-877f-11eb-8144-d7a59371cb13.png)

**Environment (If applicable):**

- OS: OSX
- Browser: Chrome
- Version: latest
- Host: Netlify / TSO (UK)

**Comment by metmarkosaric** [comment]
thanks @davestewart! Are you using Cloudflare or another proxy by any chance? 

If you're using Cloudflare for DNS, chances are you are having your sites proxied behind Cloudflare too. A proxy can override the remote IP which would lead to wrong geolocation data in your stats. Please remove the proxy and make it just a DNS record in your Cloudflare settings to solve the issue.

**Comment by davestewart** [comment]
Hey! Thanks for the info.

I'll get back to Netlify support and will find out.

I may be some time...

**Comment by metmarkosaric** [comment]
thanks Dave! i've moved this over to our discussions forum now

**Comment by davestewart** [comment]
Unfortunately, no help from Netlify, so I switched to Vercel and everything works first time.

So thanks for the input @metmarkosaric; suspect your hunch was correct.

**Reply by metmarkosaric** [reply]
thanks for the update Dave! enjoy Plausible!

**Comment by davestewart** [comment]
For future travellers, Netlify finally got back to my query; the answer is here: 

- https://answers.netlify.com/t/analytics-reporting-uk-visitors-as-us/34492

**Reply by ukutaht** [reply]
Very nice and thorough support from Netlify, props to them.

**Reply by davestewart** [reply]
Wasn't it!

**Comment by FarisZR** [comment]
is there anything else to do beside disabling cloudflare?
i had used matomo before, and it worked fine with cloudflare

**Reply by metmarkosaric** [reply]
you could try and use our proxy now that you're already using cloudflare https://plausible.io/docs/proxy/guides/cloudflare

**Reply by FarisZR** [reply]
i am self hosting plausible and our instance is behind cloudflare
so this didn't make any difference

**Reply by FarisZR** [reply]
fixed it!, the solution is here for niginx
nginx:
https://danielmiessler.com/blog/getting-real-ip-addresses-using-cloudflare-nginx-and-varnish/

apache2:
https://serverok.in/apache-cloudflare
